sdk/sdkservices/secrets.go

```diff
@@ -1,21 +0,0 @@
-package sdkservices
-
-import (
-	"context"
-)
-
-// SecretsService is a simple API for managing autokitteh user secrets.
-// So far, this is limited to connections (managed by integrations).
-type Secrets interface {
-	// Create generates a new token to represent a connection's specified
-	// key-value data, and associates them bidirectionally. If the same
-	// request is sent N times, this method returns N different tokens.
-	Create(ctx context.Context, scope string, data map[string]string, key string) (token string, err error)
-	// Get retrieves a connection's key-value data based on the given token.
```

Comment:
nit: empty line between prev func and next func comment?

Reply:
this entire file was removed.